src/benchmarklib/synthetic_table_generator.hpp

```diff
@@ -58,17 +58,31 @@ struct ColumnDataDistribution {
   double max_value;
 };
 
+struct ColumnSpecification {
+  ColumnSpecification(const ColumnDataDistribution& data_distribution, const DataType& data_type,
+                      const std::optional<std::string> name = std::nullopt,
+                      const std::optional<float> null_ratio = std::nullopt)
+      : data_distribution(data_distribution), data_type(data_type), name(name), null_ratio(null_ratio) {}
+
+  const ColumnDataDistribution data_distribution;
+
```

Comment:
Personally, I'd remove the empty lines between the declarations.

Reply:
Done in 36c31800